pkg/apis/coordination/validation/validation.go

```diff
@@ -124,29 +128,16 @@ func ValidateLeaseCandidateSpec(spec *coordination.LeaseCandidateSpec, fldPath *
 		allErrs = append(allErrs, field.Invalid(fld, spec.BinaryVersion, "must be greater than or equal to `emulationVersion`"))
 	}
 
-	if len(spec.PreferredStrategies) > 0 {
-		for i, strategy := range spec.PreferredStrategies {
-			fld := fldPath.Child("preferredStrategies").Index(i)
-
-			strategySeen := make(map[coordination.CoordinatedLeaseStrategy]bool)
-			if _, ok := strategySeen[strategy]; ok {
-				allErrs = append(allErrs, field.Duplicate(fld, strategy))
-			} else {
-				strategySeen[strategy] = true
+	if spec.Strategy != "" {
```

Comment:
`spec.Strategy` says "required", so you probably need a clause before this checking for "" and throwing `field.Required` ?